### DISCUSSION TITLE
Im on mobile today and having a hard time navigating. Is there. Version of our firmware thats working with the GD32F303 RET6

### DISCUSSION THREAD

**Discussion by Tapap84** [Jyers/Marlin#1946] [General Discussion]
It took months to convince my employer to get a printer for the shop and it has this GD RET6 chip. I dispise the stock firmware. I thought I read somewhere someone got Jyers running on it but I can't find a link

**Comment by sinkingfeeling** [comment]
Is there a plan to release a version of this firmware that works with the GD32F303 RET6 chips?

**Comment by Tapap84** [comment]
For me the STM32F103RE_creality works with gd32f. Some others had to change
environment as stated above. If you don't have a copy of the firmware you
can join the French discord and download it there. It's posted in one of
the most recent threads. Tititopher should be back soon

**Reply by sinkingfeeling** [reply]
> For me the STM32F103RE_creality works with gd32f. Some others had to change environment as stated above. If you don't have a copy of the firmware you can join the French discord and download it there. It's posted in one of the most recent threads. Tititopher should be back soon
> […](#)
> On Wed, Jul 20, 2022, 8:36 PM skarz ***@***.***> wrote: Is there a plan to release a version of this firmware that works with the GD32F303 RET6 chips? — Reply to this email directly, view it on GitHub <[#1946 (comment)](https://github.com/Jyers/Marlin/discussions/1946#discussioncomment-3193588)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ATAGNXYVEBDSEZDGECKOJN3VVCLPTANCNFSM5U44YMNA> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Which Ender are your referring to specifically?

**Comment by Tapap84** [comment]
Ender 3 v2. So far I have 4 different boards in stock ordered ender 3 v2
